lib/src/airship.dart

```diff
@@ -198,6 +198,24 @@ class Airship {
     return await _channel.invokeMethod('removeTags', tags);
   }
 
+  static Future<void> editChannelTagGroups(List<TagGroupOperation> operations) async {
```

Comment:
This interface is still not a true editor:

Ideally its this:
```
Airship.editChannelTagGroups().addTag("group", "tag").remoteTag("group", "another tag").apply()
```

Reply:
OK, so it looks like this now 
`Airship.editChannelTagGroups()..addTags("test_group", ["tag4","tag5"])..apply()`

Reply:
Oh neat - https://dart.dev/guides/language/language-tour#cascade-notation-